packages/ripple/src/utils.ts

```diff
@@ -38,17 +38,23 @@ export interface Options {
 export const colorMap: Record<Mode, Record<Variant, string>> = {
   [Mode.Light]: {
     [Variant.Primary]: visualDesignPalette.green5,
-    [Variant.Info]: transparentize(buttonOpacity, visualDesignPalette.green3),
+    [Variant.PrimaryOutline]: transparentize(
+      buttonOpacity,
+      visualDesignPalette.green3,
+    ),
     [Variant.Default]: uiColors.gray.light2,
     [Variant.Danger]: visualDesignPalette.orange4,
-    [Variant.SecondaryDanger]: transparentize(buttonOpacity, uiColors.red.base),
+    [Variant.DangerOutline]: transparentize(buttonOpacity, uiColors.red.base),
```

Comment:
We may use the ripple package for non-buttons in the future. Shouldn't these colors be defined in the Button package instead? It seems strange to define the colors here.